npm/packages/@socketsupply/socket-win32-x64/bin/install-pre-reqs.js

```diff
@@ -16,40 +16,50 @@ async function insertEnvVars () {
   }
 
   const env = JSON.parse(envJSON)
+  const sscCmdPath = path.join(process.cwd(), '../../.bin/ssc.cmd')
+
+  if (!fs.existsSync(sscCmdPath)) {
+    console.log(`WARNING: No ${sscCmdPath} builds will fail - please report (https://discord.gg/YPV32gKCsH)`)
+    return
+  }
+  let sscCmdData = fs.readFileSync(sscCmdPath, 'utf-8')
 
   let envs = ''
+  let updated = false
   for (const e in env) {
-    envs += `SET ${e}=${env[e]}\r\n`
+    const setLine = `SET ${e}=${env[e]}\r\n`
+    envs += setLine
+    // only insert or update variable if it isn't there already
+    if (sscCmdData.indexOf(setLine) === -1) {
+      if (sscCmdData.indexOf(`${e}=`) > -1) {
+        // replace if variable exists
+        sscCmdData = sscCmdData.replace(new RegExp(`${e}=.*`), `${e}=${env[e]}`)
+        updated = true
+      } else {
+        // insert variable after @ECHO off
+        sscCmdData = sscCmdData.replace('@ECHO off\r\n', `@ECHO off\r\n${setLine}`)
+        updated = true
+      }
+    }
   }
 
-  console.log(envs)
-
   fs.writeFileSync('env.bat', envs)
+  if (updated === true) { fs.writeFileSync(sscCmdPath, sscCmdData) }
 
-  const sscCmdPath = path.join(process.cwd(), '../../.bin/ssc.cmd')
-
-  if (!fs.existsSync(sscCmdPath)) {
-    console.log(`WARNING: No ${sscCmdPath} builds will fail - please report (https://discord.gg/YPV32gKCsH)`)
-    return
+  if (fs.existsSync('ps1/ps1_errors.txt')) {
+    throw fs.readFileSync('ps1/ps1_errors.txt', 'utf-8')
```

Comment:
should this `throw new Error('description')`?

Reply:
This one isn't required since we run install-pre-reqs directly from `create-socket-app`'s `index.js` (rather than in `preinstall`, `postinstall`)